### PR TITLE
Service shutdown

### PR DESCRIPTION
**Pull request by mshariq-nerd**
Implemented `/foglamp/service/shutdown`

### CODE REVIEW

File: src/app/services-health/services-health.component.ts [hunk 1]
```diff
@@ -29,23 +39,66 @@ export class ServicesHealthComponent implements OnInit {
...
+    };
+    // call child component method to toggle modal
+    this.child.toggleModal(true);
+  }
+
+  shutdownService(port) {
+    this.servicesHealthService.shutDownService(port)
+      .subscribe(
+      (data) => {
+        if (data.error) {
```
Comment (praveen-garg) [inline src/app/services-health/services-health.component.ts]: why data.error? are we not getting http error code?

File: src/app/services-health/services-health.component.ts [hunk 2]
```diff
@@ -29,23 +39,66 @@ export class ServicesHealthComponent implements OnInit {
...
+          console.log('error in response', data.error);
+          return;
+        }
+        this.alertService.success(data.message)
+        this.getServiceData();
+      },
+      (error) => {
+        console.log('error: ', error);
```
Comment (praveen-garg) [inline src/app/services-health/services-health.component.ts]: in status 0? and what if http status 400~600 error

**Comment by mshariq-nerd** [comment]
@praveen-garg  Fixed as per feedback. Please review.

File: e2e/app.po.ts [hunk 3]
```diff
@@ -86,7 +86,7 @@ export class FogLAMPPage {
 
   getServiceStatusTitle() {
     browser.ignoreSynchronization = true;
-    return element(by.css('app-services-health div:nth-child(1) strong')).getText();
+    return element(by.css('app-services-health .title.is-4')).getText();
```
Comment (praveen-garg) [inline e2e/app.po.ts]: @monika-Dianomic are you okay with this change? in panel

Comment (monika-Dianomic) [reply e2e/app.po.ts]: yes

File: src/app/services-health/services-health.component.ts [hunk 4]
```diff
@@ -29,23 +39,67 @@ export class ServicesHealthComponent implements OnInit {
       (data) => {
         /** request completed */
         this.ngProgress.done();
-        if (data.error) {  
+        if (data.error) {
```
Comment (praveen-garg) [inline src/app/services-health/services-health.component.ts]: Needs to review server side return code on error and handle that. NOT in 200

**Pull request merged**